opentelemetry-api/pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
 dependencies = [
     "Deprecated >= 1.2.6",
     "setuptools >= 16.0",
-    "importlib-metadata >= 5.0.0; python_version=='3.7'"
+    "importlib-metadata >= 5.0.0",
```

Comment:
You might want to add an upper bound since they keep making breaking changes. (we can expand the upper bound as long as the API is compatible). Seems like the 6.0 release works fine right?

Also, can you add a comment here that this can be dropped after 3.11?

Reply:
Done